src/test/java/org/cactoos/TextTest.java

```diff
@@ -37,12 +38,12 @@
 
     @Test(expected = IllegalArgumentException.class)
     public void failForNullArgument() throws Exception {
-        new Text.NoNulls(null).asString();
+        new NoNulls(null).asString();
     }
 
     @Test(expected = IllegalStateException.class)
```

Comment:
@dgroup The same above.

Reply:
@fabriciofx disagree, https://github.com/yegor256/cactoos/pull/990#discussion_r247960573